numcodecs/compat_ext.pyx

```diff
@@ -1,19 +1,16 @@
-# -*- coding: utf-8 -*-
 # cython: embedsignature=True
 # cython: profile=False
 # cython: linetrace=False
 # cython: binding=False
-# cython: language_level=2
-from __future__ import absolute_import, print_function, division
+# cython: language_level=3
 from cpython.buffer cimport PyObject_GetBuffer, PyBuffer_Release
 
 
 from .compat import ensure_contiguous_ndarray
 
 
 cdef class Buffer:
-    """Compatibility class to work around fact that array.array does not support
-    new-style buffer interface in PY2."""
+    """Convenience class for buffer interface"""
```

Comment:
```suggestion
    """Convenience class for buffer interface."""
```

Thanks for updating this docstring. Apologies, I am a total obsessive when it comes to punctuation :-)

Reply:
No problem : )